modules/mono/glue/GodotSharp/GodotSharp/Core/AABB.cs

```diff
@@ -145,6 +145,9 @@ public real_t GetArea()
         /// Gets the position of one of the 8 endpoints of the <see cref="AABB"/>.
         /// </summary>
         /// <param name="idx">Which endpoint to get.</param>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// Thrown when the given <paramref name="idx"/> is not a value between 0 and 7 (inclusive).
```

Comment:
The .NET docs seem to follow a similar style to exception messages (no `thrown when`). They also just say the parameter name directly instead of something like `The given parameter`. So this could be:

"idx is not a value between 0 and 7 (inclusive)."

Additionally, I think this is easier read as:

"idx is less than 0 or greater than 7."

They have a special way to separate this by the -or-:

- https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.item?view=net-6.0#exceptions
- https://github.com/dotnet/dotnet-api-docs/blob/6019bf2f182c372ca998f95b7d6d10e6bac3c038/xml/System.Collections.Generic/List%601.xml#L3459-L3463